### DISCUSSION TITLE
A brief MusE history

### DISCUSSION THREAD

**Discussion by spamatica** [muse-sequencer/muse#1202] [Team Posts]
As I was suspecting MusE passed 20 years last year I tried doing some digital archeology at that time, to find really old versions of MusE, there was supposed to be a muse-0.0.1 release. Unfortunately I didn't find any traces of it at the time so I never did anything more with it.

However, earlier today I suddenly remembered to check my own backups and as it turns out I was particularly good at burning cd backups around the millenium. 
Going through them I was for starters amazed that they all seem to work still. And what do you know, being a bit of a digital hamster I found a bunch of really old MusE versions, including muse-0.0.1.tar.bz2 :)

https://spamatica.com/muse-0.0.1.tar.bz2

I think I will try to upload them all somewhere for public access (though the morale of the story is that free online services should not  be treated as a permanent backup).

**Comment by terminator356** [comment]
Oh the things lying around on my many hard drives from decades ago...

I'll get to work right away merging the code ;-)
Hm, why does the folder tree inside say muse-1.1.1?

I have searched before as well, and the earliest I found online was 0.6.1
There is another project called muse (streaming app), and I was fooled once by
 finding their very old code, I thought it was old lmuse code but it wasn't.

**Comment by spamatica** [comment]
Ah, I didn't see that the actual folder name was 1.1.1. 
I'm pretty sure it is the authentic 0.0.1 release though, as the package was created on the 10th of january 2000 and the Changelog only has entries from 1999 (in german).
It is safe to say, a lot has changed since then :)

I did the same mistake btw found 0.0.1 on the internet but it was the 'other' muse project. Well not it is found anyway!
